matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/crypto/model/MXUsersDevicesMap.kt

```diff
@@ -46,8 +46,8 @@ class MXUsersDevicesMap<E> {
     /**
      * Provides the object for a device id and a user Id.
      *
-     * @param deviceId the device id
      * @param userId   the object id
```

Comment:
should be `* @param userId            the user id` instead of  `* @param userId   the object id`

Reply:
good catch yes.